internal/cmd/stack/list.go

```diff
@@ -18,20 +21,57 @@ func listStacks() cli.ActionFunc {
 			return err
 		}
 
+		var limit *uint
+		if cliCtx.IsSet(flagLimit.Name) {
+			if cliCtx.Uint(flagLimit.Name) == 0 {
+				return fmt.Errorf("limit must be greater than 0")
+			}
+
+			if cliCtx.Uint(flagLimit.Name) >= math.MaxInt32 {
```

Comment:
Are such checks really necessary? No point in removing it now, but also eeh 🤷🏻

Reply:
I like it :) Validation is helpful

Reply:
It might be a matter of taste, but I like knowing what to expect from inputs  :)